moorhuhn-service/src/main/java/com/moorhuhnservice/moorhuhnservice/BaseClasses/Question.java

```diff
@@ -0,0 +1,96 @@
+package com.moorhuhnservice.moorhuhnservice.BaseClasses;
+
+
+import javax.persistence.*;
+@Entity
+@Table(name = "questions")
+public class Question {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id", nullable = false)
+    private Long id;
+    @Column(nullable = false)
+    private String level;

```

Comment:
I'm not quite sure what this is, and why it is needed...

Reply:
this is the identification string that the right questions are loaded in the right level, if you have 2 moorhuhn games in different levels you only want the questions that are for this level.